cypress/e2e/workspace.spec.js

```diff
@@ -43,6 +43,32 @@ describe('Workspace', function() {
 		})
 	})
 
+	it('Hides the workspace when switching to another folder', function() {
+		cy.uploadFile('test.md', 'text/markdown', `${Cypress.currentTest.title}/README.md`)
+		cy.createFolder(`${Cypress.currentTest.title}/subdirectory`)
+		cy.reload()
+		cy.get('.files-fileList').should('contain', 'README.md')
+		cy.get('#rich-workspace .ProseMirror')
+			.should('contain', 'Hello world')
+		cy.openFolder('subdirectory')
+		cy.get('#rich-workspace')
+			.get('.ProseMirror')
```

Comment:
The first get already fails. So i guess you want this:
```suggestion
		cy.get('#rich-workspace .ProseMirror')
```